### PR TITLE
CMake update after Eyescale/CMake#341

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -38,6 +66,7 @@ add_subdirectory(tests)
 # Documentation
 set(COMMON_PROJECT_DOMAIN ch.epfl.bluebrain)
 include(DoxygenRule)
+include(PackageConfig)
```
Comment (rdumusc) [inline CMakeLists.txt]: Hmm, do we add PackageConfig to all the applications? It is not really a reusable library, I don't think users want to do a find_package(DisplayCluster)...

**Comment by rdumusc** [comment]
+1

File: CMakeLists.txt (hunk 1 above)
Comment (tribal-tec) [inline CMakeLists.txt]: It's needed for an integration into a superproject. Otherwise, DisplayCluster cannot be found, while doing git_subproject().

Comment (rdumusc) [inline CMakeLists.txt]: Go for it! Although I don't understand the problem... A project which is not found is cloned, and this is what you want anyway if you put a top-level project inside a meta-project, no? If that's not the case, we have to document this in Subproject.cmake.

**Comment by tribal-tec** [comment]
retest this please

**Comment by tribal-tec** [comment]
retest this please

**Comment by tribal-tec** [comment]
retest this please

**Pull request merged**